.ruby-version

```diff
@@ -0,0 +1 @@
+2.2.2
```

Comment:
@chrisruenes1 if you rebase this branch off `master` this problem will go away. I ignored the .ruby-version file altogether to avoid this problem now and forever.